### PR TITLE
exercism.org

### PR DESCRIPTION
**Pull request by MenkeTechnologies**

### CODE REVIEW

**Review by NobbZ — CHANGES_REQUESTED** [review]
The links all go through a redirect, can you please "resolve" them such that the links in the files do not need to through any redirects?

File: exercises/practice/hello-world/hello_world_tests.plt [hunk 1]
```diff
@@ -1,6 +1,6 @@
-% Please visit https://exercism.io/tracks/prolog/installation
+% Please visit https://exercism.org/tracks/prolog/installation
```
Comment (ErikSchierboom) [inline exercises/practice/hello-world/hello_world_tests.plt:1]:
```suggestion
% Please visit https://exercism.org/docs/tracks/prolog/installation
```

File: exercises/practice/hello-world/hello_world_tests.plt [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 % for instructions on setting up prolog.
-% Visit https://exercism.io/tracks/prolog/tests
+% Visit https://exercism.org/tracks/prolog/tests
```
Comment (ErikSchierboom) [inline exercises/practice/hello-world/hello_world_tests.plt:3]:
```suggestion
% Visit https://exercism.org/tracks/docs/prolog/tests
```

File: exercises/practice/queen-attack/queen_attack.pl [hunk 3]
```diff
@@ -1,6 +1,6 @@
-% Please visit https://exercism.io/tracks/prolog/installation
+% Please visit https://exercism.org/tracks/prolog/installation
```
Comment (ErikSchierboom) [inline exercises/practice/queen-attack/queen_attack.pl:1]:
```suggestion
% Please visit https://exercism.org/docs/tracks/prolog/installation
```

File: exercises/practice/queen-attack/queen_attack.pl [hunk 4]
```diff
@@ -1,6 +1,6 @@
...
 % for instructions on setting up prolog.
-% Visit https://exercism.io/tracks/prolog/tests
+% Visit https://exercism.org/tracks/prolog/tests
```
Comment (ErikSchierboom) [inline exercises/practice/queen-attack/queen_attack.pl:3]:
```suggestion
% Visit https://exercism.org/docs/tracks/prolog/tests
```

File: exercises/practice/queen-attack/queen_attack_tests.plt [hunk 5]
```diff
@@ -1,6 +1,6 @@
- % Please visit https://exercism.io/tracks/prolog/installation
+ % Please visit https://exercism.org/tracks/prolog/installation
```
Comment (ErikSchierboom) [inline exercises/practice/queen-attack/queen_attack_tests.plt:1]:
```suggestion
% Please visit https://exercism.org/docs/tracks/prolog/installation
```

File: exercises/practice/queen-attack/queen_attack_tests.plt [hunk 6]
```diff
@@ -1,6 +1,6 @@
...
 % for instructions on setting up prolog.
-% Visit https://exercism.io/tracks/prolog/tests
+% Visit https://exercism.org/tracks/prolog/tests
```
Comment (ErikSchierboom) [inline exercises/practice/queen-attack/queen_attack_tests.plt:3]:
```suggestion
% Visit https://exercism.org/docs/tracks/prolog/tests
```

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**